amundsen_application/static/js/components/SearchPage/index.tsx

```diff
@@ -6,44 +6,63 @@ import Pagination from 'react-js-pagination';
 import SearchBar from './SearchBar';
```

Comment:
nit nit: Our mix of having variables in camelCase (JS standard) vs snake_case (Python standard) in our React code is something I'd like to avoid making worse. In the future can we move closer to camelCase or at least prevent mixing in more snake_case? 

Not sure what the best approach is though. I recognize it's a pretty tedious/low-priority task to parse all of our responses to create objects with all camelCase variables, but the consistency will be nice and may help when we want to start enforcing stricter lint rules.

Reply:
I too, dislike the mixing of different standards, but converting nested fields for every data payload seems onerous.

One convention that we could follow is to use camelCase for all functions and variables defined in JS, but preserve snake_case for any data that we receive.

Reply:
I just noticed that my find-and-replace also hit JS defined variables too. Will fix.